english/coffee/hollywood/hollywood.md

```diff
@@ -1,3 +1,5 @@
 After writing, what you believe to be to be, a best-selling novel you decided to try to market the book right
-and head to Hollywood. The only choice now is how to get there. You’re afraid of flying and don’t care much for 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.